### PR TITLE
Fix header alignment

### PR DESCRIPTION
**Pull request by wadeking98**
# Summary of Changes

- Aligned all title headers to the center
- Removed Text from back arrows

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/screens/ProofRequestAttributeDetails.tsx [hunk 1]
```diff
@@ -60,13 +60,13 @@ const ProofRequestAttributeDetails: React.FC<ProofRequestAttributeDetailsProps>
       headerRight: undefined,
       headerLeft: () => (
         <HeaderLeftBack
-          title={t('Global.Back')}
+          title={' '}
```
Comment (amanji) [inline core/App/screens/ProofRequestAttributeDetails.tsx:63]: Was part of the ticket removing the back buttons?

Comment (wadeking98) [reply core/App/screens/ProofRequestAttributeDetails.tsx:63]: not officially, but the kim wanted the back button text removed

Comment (amanji) [reply core/App/screens/ProofRequestAttributeDetails.tsx:63]: In that case do we need the title? Or will it display the default? If we need to use empty strings we should probably use have no spaces.

File: core/App/navigators/ContactStack.tsx [hunk 2]
```diff
@@ -12,12 +11,11 @@ import { createDefaultStackOptions } from './defaultStackOptions'
 
 const ContactStack: React.FC = () => {
   const Stack = createStackNavigator<ContactStackParams>()
-  const { t } = useTranslation()
   const theme = useTheme()
   const defaultStackOptions = createDefaultStackOptions(theme)
   return (
     <Stack.Navigator screenOptions={{ ...defaultStackOptions }}>
-      <Stack.Screen name={Screens.Contacts} component={ListContacts} options={{ headerBackTitle: t('Global.Back') }} />
+      <Stack.Screen name={Screens.Contacts} component={ListContacts} options={{ headerBackTitle: ' ' }} />
```
Comment (amanji) [inline core/App/navigators/ContactStack.tsx]: Might be better to use the `headerBackTitleVisible` option here and remove the `headerBackTitle` option altogether.

**Review by amanji — CHANGES_REQUESTED** [review]

**Comment by JamesKEbert** [comment]
@ryanindicio could you review?

File: core/App/screens/ProofRequestAttributeDetails.tsx (hunk 1 above)
Comment (wadeking98) [reply core/App/screens/ProofRequestAttributeDetails.tsx:63]: it displays default without the title, but also when the title is set to an empty string for some reason. That's why I had to set it to be a space

File: core/App/navigators/ContactStack.tsx (hunk 2 above)
Comment (wadeking98) [reply core/App/navigators/ContactStack.tsx]: Okay I changed it to `headerBackTitleVisible: false`

File: core/App/screens/ProofRequestAttributeDetails.tsx [hunk 3]
```diff
@@ -60,13 +60,13 @@ const ProofRequestAttributeDetails: React.FC<ProofRequestAttributeDetailsProps>
...
           onPress={() => {
             navigation.pop()
           }}
         />
       ),
-      headerBackTitle: t('Global.Back'),
+      headerBackTitleVisible: false,
       headerBackAccessibilityLabel: t('Global.Back'),
```
Comment (amanji) [inline core/App/screens/ProofRequestAttributeDetails.tsx]: Can remove the a11y label if we're not displaying anything

**Review by amanji — CHANGES_REQUESTED** [review]

File: core/App/screens/ProofRequestAttributeDetails.tsx (hunk 3 above)
Comment (wadeking98) [reply core/App/screens/ProofRequestAttributeDetails.tsx]: okay got it, I wasn't sure if it was needed for screen readers

Comment (wadeking98) [reply core/App/screens/ProofRequestAttributeDetails.tsx]: change completed

**Review by jleach — APPROVED** [review]

**Review by jleach — APPROVED** [review]

**Review by amanji — APPROVED** [review]

**Pull request merged**